### DISCUSSION TITLE
`dateTimeShift` acts strange in adding days

### DISCUSSION THREAD

**Discussion by ArsalanImamverdi** [mockoon/mockoon#1296] [Help]
I need to add days to `now` for example 5 days, if the date is `2024-01-30`, then the result is `2024-01-35`!
`"DepartureDateTime": "{{dateTimeShift date=(now) format='YYYY-MM-DD' days=5}} 20:00"`

**Comment by 255kb** [comment]
I was scratching my head to find why I got it working with other examples from the docs: 

```
{{dateTimeShift date='2024-02-02' format='yyyy-MM-dd' days=40}}
{{dateTimeShift date=(now) format='yyyy-MM-dd' days=40}}

--> 2024-03-13 
--> 2024-03-13 
```

You are using 'DD' in the format which are the days of the year. You should use 'dd' instead. 
All our date helpers are using [date-fns format function](https://date-fns.org/v2.30.0/docs/format). I will make it clearer in the docs.
